packages/alchemy/src/middleware/gasManager.ts

```diff
@@ -190,67 +185,146 @@ export function alchemyGasManagerMiddleware<C extends ClientWithAlchemyMethods>(
   };
 }
 
+const overrideField = <TEntryPointVersion extends EntryPointVersion>(
+  field: keyof UserOperationFeeOptions<TEntryPointVersion>,
+  overrides: UserOperationOverrides<TEntryPointVersion> | undefined,
+  feeOptions: UserOperationFeeOptions<TEntryPointVersion> | undefined,
+  userOperation: UserOperationRequest<TEntryPointVersion>
+): Hex | Multiplier | undefined => {
+  let _field = field as keyof UserOperationOverrides<TEntryPointVersion>;
+
+  if (overrides?.[_field] != null) {
+    // one-off absolute override
+    if (isBigNumberish(overrides[_field])) {
+      return deepHexlify(overrides[_field]);
+    }
+    // one-off multiplier overrides
+    else {
+      return {
+        multiplier: Number((overrides[_field] as Multiplier).multiplier),
+      };
+    }
+  }
+
+  // provider level fee options with multiplier
+  if (isMultiplier(feeOptions?.[field])) {
+    return {
+      multiplier: Number((feeOptions![field] as Multiplier).multiplier),
+    };
+  }
+
+  const userOpField =
+    userOperation[field as keyof UserOperationRequest<TEntryPointVersion>];
+  if (isHex(userOpField) && fromHex(userOpField as Hex, "bigint") > 0n) {
+    return userOpField;
+  }
+  return undefined;
+};
+
 function requestGasAndPaymasterData<C extends ClientWithAlchemyMethods>(
   client: C,
   config: AlchemyGasManagerConfig
 ): ClientMiddlewareConfig["paymasterAndData"] {
   return {
     dummyPaymasterAndData: dummyPaymasterAndData(client, config),
-    paymasterAndData: async (struct, { overrides, feeOptions, account }) => {
+    paymasterAndData: async (
+      struct,
+      { overrides: overrides_, feeOptions, account }
+    ) => {
+      // handle one-off overrides for bypassing paymaster middleware
+      if (
+        account.getEntryPoint().version === "0.6.0" &&
+        (overrides_ as UserOperationOverrides<"0.6.0">)?.paymasterAndData ===
+          "0x"
```

Comment:
should be use `EmptyHex` here?

Reply:
That's not a thing at the moment. `EmptyHex` is a type, not a value.

If you're suggesting creating a constant for it, e.g.
```ts
export const EMPTY_HEX: EmptyHex = "0x";
```
I dunno if I think it improves readability really, but I wouldn't be opposed if other people like it.

Reply:
ah gotcha, misread the file. thanks!